src/components/Flipper/FlipCustomize/FlipCustomize.tsx

```diff
@@ -191,10 +207,22 @@ function FlipCustomize() {
         window.location.reload();
     }
 
+    function getDefaultValues() {
+        return FLIP_FINDERS.filter(option => settings.finders?.some(finder => finder.toString() === option.value))
+    }
+
     let useLowestBinHelpElement = (
         <p>By enabling this setting, the lowest BIN is used as the estimated selling price to calculate your profit. That can lead to profitable flips being estimated way too low (even as a loss). We recommend using the median to calculate the profit.</p>
     );
 
+    let flipFindersElement = (
+        <ul>
+            <li><b>Flipper </b>is the classical flip finding algorithm using the Skyblock AH history database. It searches the history for similar items but searching for references takes time thus this is relatively slow.</li>
```

Comment:
could these tooltips be displayed on the actual element itself? 
![image](https://user-images.githubusercontent.com/16632490/144648867-18817325-5947-43b9-8efe-6fbb97a3e82a.png)
It will probably grow in the not to distant future and is already very hard to read